javascript/express/security/injection/tainted-sql-string.yaml

```diff
@@ -36,6 +36,7 @@ rules:
                   - metavariable-regex:
                       metavariable: $METHOD
                       regex: ^(get|post|put|head|delete|options)
+          - focus-metavariable: $REQ
```

Comment:
Hmm, in this case this won't have the effect you want, you're kind of saying that `$REQ` is tainted rather than `$REQ.query`. So let's say that the code uses `req.query`, and later on you use `req.abc`, I think adding this `focus-metavariable` would cause `req.abc` to be tainted because you're specifying that `req` itself is tainted. In short, I would drop this line.